src/model/press-package.v1.yaml

```diff
@@ -0,0 +1,43 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Press package
+type: object
+allOf:
+  - $ref: ../snippets/press-package.v1.yaml
+  - properties:
+        content:
+            description: Content
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        relatedContent:
+            title: Related content
+            type: array
+            items:
+                oneOf:
+                  - $ref: ../snippets/article-poa.v1.yaml
+                  - $ref: ../snippets/article-vor.v1.yaml
+            minItems: 1
+            uniqueItems: true
+        mediaContacts:
+            title: Media contacts
+            type: array
+            items:
+                allOf:
+                  - $ref: ../misc/person.v1.yaml
+                  - properties:
+                      emailAddresses:
+                          type: array
+                          items:
+                              type: string
+                              format: email
+                  phoneNumbers:
+                      type: array
+                      items:
+                          $ref: phone-number.v1.yaml
+                  postalAddresses:
+                      type: array
+                      items:
+                          $ref: address.v1.yaml
+    required:
+      - content
```

Comment:
I suggest we make `relatedContent` required. If we later find out it's optional we can always change it.